src/ucp/dt/dt.c

```diff
@@ -49,26 +49,24 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_mem_type_unpack,
                                       mem_type, md_index, memh, &md_map,
                                       &rkey_bundle);
     if (status != UCS_OK) {
-        ucs_error("failed to register buffer with mem type domain %s",
+        ucs_fatal("failed to register buffer with mem type domain %s",
                   ucs_memory_type_names[mem_type]);
-        return status;
     }
 
     status = uct_ep_put_short(ep->uct_eps[lane], recv_data, recv_length,
                               (uint64_t)buffer, rkey_bundle.rkey);
     if (status != UCS_OK) {
-        ucs_error("uct_ep_put_short() failed %s", ucs_status_string(status));
+        ucs_fatal("uct_ep_put_short() failed %s", ucs_status_string(status));
```

Comment:
need to print it's related to mem type unpack

Reply:
as I can see ucs_fatal macro adds `__FILE__`,` __LINE__` and `__FUNCTION__` to output. do we need additional note about mem type unpack?

Reply:
yes, let's make it even more clear

Reply:
ok, updated